conda-environments/DEEPLABCUT_PPC.yaml

```diff
@@ -0,0 +1,31 @@
+name: DEEPLABCUT_PPC
+channels:
+  - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+  - powerai
+  - defaults
+  - conda-forge
+  - hcc
+dependencies:
```

Comment:
DLC is not a dependency?

Reply:
Hi! Sorry, my bad - removed it for testing. DLC is in again.